src/signed_data.rs

```diff
@@ -352,6 +372,36 @@ const RSA_PKCS1_SHARED: PublicKeyAlgorithmSharedInfo =
     allowed_signature_alg_parameters: &[&[], &[0x05, 0x00]], // Optional NULL.
 };
 
+const RSA_PSS_SHA256: PublicKeyAlgorithm = PublicKeyAlgorithm {
+    shared: &PublicKeyAlgorithmSharedInfo {
```

Comment:
This whole scheme doesn't really makes less sense now, but it's not your fault. I guess we should refactor how all this is done sometime soon.